science/vcftools/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            vcftools vcftools 0.1.16 v
+github.tarball_from     releases
+revision                0
+checksums               rmd160  2ddef86b8fa2c325bb063be0843e35655b83c932 \
+                        sha256  dbfc774383c106b85043daa2c42568816aa6a7b4e6abc965eeea6c47dde914e3 \
+                        size    480575
+
+categories              science
+platforms               darwin
+maintainers             nomaintainer
+license                 GPL-3
+
+description             utilities for working with VCF files
+
+long_description        A set of tools written in Perl and C++ for working with \
+                        VCF files, such as those generated by the 1000 Genomes \
+                        Project.
+
+homepage                https://vcftools.github.io/
+
+depends_build           pkgconfig
+depends_lib             path:bin/perl:perl5
```

Comment:
Please use 

depends_lib-append 

appending is always correct, whereas not runs the risk of wiping out previously added deps, so its good practise to just aways append